tests/scripts/pulpcore/test_content_guards.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# shellcheck source=tests/scripts/config.source
+. "$(dirname "$(dirname "$(realpath "$0")")")"/config.source
+
+pulp debug has-plugin --name "core" --min-version "3.15.1" || exit 3
+
+cleanup() {
+  pulp content-guard rbac destroy --name "cli_test_guard" || true
+  pulp group destroy --name "cli_test_group" || true
+}
+trap cleanup EXIT
+
+expect_succ pulp content-guard rbac create --name "cli_test_guard"
+expect_succ pulp content-guard list
+test "$(echo "$OUTPUT" | jq -r length)" -gt "0"
+expect_succ pulp content-guard rbac list --name "cli_test_guard"
+test "$(echo "$OUTPUT" | jq -r length)" -eq "1"
+expect_succ pulp content-guard rbac show --name "cli_test_guard"
+
+expect_succ pulp group create --name "cli_test_group"
+expect_succ pulp content-guard rbac assign --name "cli_test_guard" --groups '["cli_test_group"]'
```

Comment:
From a usability perspective, do you think it would be useful to have "--user" and "--group" parameters with the `multiple=True` flag? This should not change the code much, but the user would not need to write json.
(Most of the time you probably want to add one user or group.)